### PR TITLE
create clear functionality

### PR DESCRIPTION
**Pull request by ljang0**

### CODE REVIEW

File: browser_env/actions.py [hunk 1]
```diff
@@ -1319,6 +1362,16 @@ async def aexecute_action(
             await aexecute_mouse_click(
                 action["coords"][0], action["coords"][1], page
             )
+        case ActionTypes.CLEAR:
+            element_id = action["element_id"]
+            element_center = obseration_processor.get_element_center(element_id)  # type: ignore[attr-defined]
+            execute_mouse_click(element_center[0], element_center[1], page)
+            execute_key_press("Meta+A", page)
+            execute_key_press('Backspace', page)
+        case ActionTypes.CLEAR:
```
[the remark this answers is not in the document]
Comment (ljang0) [reply browser_env/actions.py]: just got rid of that typo sorry - wasn't getting caught during testing

File: browser_env/actions.py [hunk 2]
```diff
@@ -308,6 +322,7 @@ class ActionTypes(IntEnum):
     CHECK = 15
     SELECT_OPTION = 16
 
+    CLEAR = 16
```
Comment (kohjingyu) [inline browser_env/actions.py:325]: We need to give CLEAR a unique ID, what about using 18 instead? 16 is the SELECT_OPTION action.

**Review by kohjingyu — CHANGES_REQUESTED** [review]

File: browser_env/actions.py (hunk 2 above)
Comment (ljang0) [reply browser_env/actions.py]: just changed it to 18

**Review by kohjingyu — APPROVED** [review]
Thanks!

**Comment by ljang0** [comment]
fixed one more bug - forgot to use await in aexecute_action.

**Review by kohjingyu — APPROVED** [review]

**Pull request merged**